src/net_processing.cpp

```diff
@@ -250,6 +250,9 @@ struct Peer {
       * to the peer. */
     std::chrono::microseconds m_next_send_feefilter{0};
 
+    //! Length of current-streak of unconnecting headers announcements
+    int num_unconnecting_headers_msgs{0};
```

Comment:
Should this be named `m_num_unconnecting_headers_msgs` for consistency with the style guide?

Reply:
`Peer` is a `struct`, so it shouldn't use `m_` prefixes. To quote the style guide:

> Class member variables have a m_ prefix.

For rationale, see for example https://github.com/bitcoin/bitcoin/pull/19759

(So in theory https://github.com/bitcoin/bitcoin/pull/25557 could drop the `m_` prefix)

Though, I don't care about the style guide too much, so I am happy to change this to make reviewers happy and I am also not going to comment on https://github.com/bitcoin/bitcoin/pull/25557 to suggest a change.